### DISCUSSION TITLE
Is there any way to mute a character?

### DISCUSSION THREAD

**Discussion by ainaerco** [Lurkars/gloomhavensecretariat#110] [Q&A] [answered]
In Frosthaven campaign we may have occasional players that come and go. Therefore not all characters could be present on particular scenario. Is there any way to mute a character temporarily instead of deleting?

**Comment by Lurkars** [comment, marked as answer]
Hey, yes there is!
From [F.A.Q.](https://help.gloomhaven-secretariat.de/#faq)

> **A player is missing for a session, how to keep it's character in the party without participating in scenario?**
There is a Absent feature implemented. After started the scenario, mark a character as absent by opening the [character sheet](https://help.gloomhaven-secretariat.de/#characrer-sheet) and just click on the character name. The name should be striked-through to indicate its absent. To hide absent characters in the list, use the setting **Hide absent figures**.

I just added recently, that you can also click on the characters name in the characters menu!

**Reply by ainaerco** [reply]
Nice, thank you!
